src/models/Review.py

```diff
@@ -26,7 +26,7 @@ def toJSON(self) -> dict:
     # convert _id fields and date
     if self._id is not None:
       res['_id'] = str(self._id) 
-    res['user_id'] = str(self.user_id)
+    res['user_email'] = str(self.user_email)
```

Comment:
In hindsight, this isn't needed. Since `user_email` is a string, it's automatically handled by the `vars()` a few lines earlier. Can you remove this please?